indico/modules/rb_new/client/js/common/bookings/selectors.js

```diff
@@ -28,3 +30,6 @@ export const getDetailsWithRoom = createSelector(
     }
 );
 export const hasDetails = (state, {bookingId}) => getDetails(state, {bookingId}) !== undefined;
+export const isBookingChangeInProgress = ({bookings: {requests}}) => (
```

Comment:
usually we only destructurize the top-level state slice and use the usual dot notation for the rest in selectors